ui/sections/storage/inspectors/disk.reel/disk.js

```diff
@@ -14,6 +14,32 @@ exports.Disk = AbstractInspector.specialize({
                     value: x
                 };
             });
+            this.addPathChangeListener('object.path', this, '_handleDiskChange');
+        }
+    },
+
+    _handleDiskChange: {
```

Comment:
Latest change in storage-repository make this one useless (sorry about that). Now, you can just use `object.isAvailable` as a flag to know if a disk is used by a volume or not.